dashboard/client/src/components/ActorTable.tsx

```diff
@@ -40,6 +43,102 @@ const ActorTable = ({
   const list = actorList.slice((pageNo - 1) * pageSize, pageNo * pageSize);
   const classes = rowStyles();
 
+  const columns = [
+    { label: "" },
+    { label: "ID" },
+    {
+      label: "Class",
+      helpInfo: (
+        <Typography>
+          The class name of the actor. For example, the below actor has a class
+          name `Actor`.
+          <br />
+          <br />
+          @ray.remote
+          <br />
+          class Actor:
+          <br />
+          &emsp;pass
+          <br />
+        </Typography>
+      ),
+    },
+    {
+      label: "Name",
+      helpInfo: (
+        <Typography>
+          The name of the actor given by an `name` argument. For example, this
+          actor's name is `unique_name`.
+          <br />
+          <br />
+          Actor.options(name="unique_name").remote()
+        </Typography>
+      ),
+    },
+    {
+      label: "State",
+      helpInfo: (
+        <Typography>
+          The state of the actor. States are documented at a `ActorState` from a
+          `gcs.proto` file.
```

Comment:
```suggestion
          The state of the actor. States are documented as "ActorState" in the
          "gcs.proto" file.
```

Reply:
@rkooo567 @alanwguo 
Shall we actually document the "states" of actors, nodes, etc. in the ray documentation? Probably in the documentation of dashboard or state api later? Maybe in 2.3 or + after we finalize the list of states.

Reply:
Yeah sgtm cc @rickyyx  to record the work items!